lib/ddtrace/contrib/redis/patcher.rb

```diff
@@ -59,10 +59,10 @@ def call_pipeline(*args, &block)
               pin.tracer.trace(Datadog::Contrib::Redis::Ext::SPAN_COMMAND) do |span|
                 span.service = pin.service
                 span.span_type = Datadog::Contrib::Redis::Ext::TYPE
-                commands = args[0].commands.map { |c| Datadog::Contrib::Redis::Quantize.format_command_args(c) }
+                commands = get_pipeline_commands(args)
```

Comment:
@marcotc Assignment Branch Condition size has been violated because of this change: do you normally extract these out or ignore the cop?

Reply:
Your call, to be honest. I'm ok with either approach.

I noticed you extracted it, which actually looks much nicer!

Reply:
Sorry @marcotc the issue is in `patch_redis_client` (which does the `class_eval`).

Reply:
Go ahead and add a "Assignment Branch Condition" exception here, no worries.

Reply:
Done, thank you @marcotc